CorePayments/src/main/java/com/paypal/android/corepayments/features/eligibility/EligibilityClient.kt

```diff
@@ -0,0 +1,53 @@
+package com.paypal.android.corepayments.features.eligibility
+
+import android.content.Context
+import com.paypal.android.corepayments.CoreConfig
+import com.paypal.android.corepayments.PayPalSDKError
+import com.paypal.android.corepayments.apis.eligibility.EligibilityAPI
+import kotlinx.coroutines.CoroutineDispatcher
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.launch
+
+/**
+ *  Use EligibilityClient to check eligibility for different payment methods.
+ */
+class EligibilityClient internal constructor(
```

Comment:
Can we add unit test coverage for this class?

Reply:
Oh yeah I forgot to write the unit test for this one. I'll add it in another commit.

Reply:
Added unit tests here [3e27393](https://github.com/paypal/paypal-android/pull/268/commits/3e273930acd828d28aafeecb3a92fcda866467c5) and here [5ddb367](https://github.com/paypal/paypal-android/pull/268/commits/5ddb367e7624f4ef3378d58806438435e96f9b69).